docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -955,6 +993,41 @@ that this value is normally distributed, but we could use the GBD
 ensemble risk exposure machinary if that assumption seems like a
 limitation.
 
+2.3.13 Employment (18)
+~~~~~~~~~~~~~~~~~~~~~~
+
+To represent businesses and employment dynamics we will use another
+directed tripartite graph (analogous to our migration component),
+showing arcs from simulants (part A) to employers (part B) as well as
+arcs from employers to their addresses (part C).
+
+This construct allows us to represent businesses that employ one or
+more people, as well as individuals who are employed by multiple
+businesses.  We will also be able to add business dynamics in the
+future, e.g. new businesses arriving, old businesses closing down, and
+even merges, as well as name changes and address changes.  All of this
+will go into our simulated tax return data, which we must make a
+scheme for before we access restricted tax data (since even the schema
+of this data is restricted information).
+
+To keep things simple for starters, we will give everyone over 18 a
+random edge to an employer, chosen from a skewed distribution to
+ensure that there are a few large employers and a "fat tail" of small
+employers. We will change employers randomly at the rate of 50 changes
+per 100 person years, and change employer addresses at a rate of 10
```

Comment:
if i make one of the employers "unemployed", (i can google florida unemployment rate), we will also want rate at which unemployed people become employed, and rate at which employed people become unemployed. (i can make these up for now if you dont have data)

Reply:
ok, good idea.